src/Task/PhpCsFixer.php

```diff
@@ -71,11 +71,12 @@ public function run(ContextInterface $context): TaskResultInterface
         $this->formatter->resetCounter();
 
         $arguments = $this->processBuilder->createArgumentsForCommand('php-cs-fixer');
-        $arguments->add('--format=json');
+        $arguments->add('--format=txt');
```

Comment:
I think this task is using a dedicated formatter (GrumPHP\Formatter\PhpCsFixerFormatter), that this change will break

Reply:
Indeed, this will bypass the custom formatter.
However, the formatter is quite old and was used to propose a way of fixing the code.
Now that we have our fixer configuration - which used the process to build de fixer process - it seems safe to nuke that custom formatter. It doesn't really provide any benefits anymore + it seems better to rely on the official php-cs-fixer formatters.

@oallain : Can you:

- remove the custom formatter and it's implementation
- provide a configurable `format` string option with a default of `txt`
- provide a configurable `ansii` boolean option with a default of `true`.
- add documentation for these options?